packages/headless/ssr/package.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
- [ ] @louis-bompart check this later, :suspect: monorepo structure (3rd level pjson)